homeassistant/components/systemmonitor/sensor.py

```diff
@@ -60,62 +66,188 @@
-        PERCENTAGE,
-        "mdi:memory",
-        None,
-        False,
+    "throughput_network_in": SysMonitorSensorEntityDescription(
+        key="throughput_network_in",
+        name="Network throughput in",
+        native_unit_of_measurement=DATA_RATE_MEGABYTES_PER_SECOND,
+        icon="mdi:server-network",
+        state_class=STATE_CLASS_TOTAL,
+        mandatory_arg=True,
     ),
-    "network_in": ("Network in", DATA_MEBIBYTES, "mdi:server-network", None, True),
-    "network_out": ("Network out", DATA_MEBIBYTES, "mdi:server-network", None, True),
-    "packets_in": ("Packets in", " ", "mdi:server-network", None, True),
-    "packets_out": ("Packets out", " ", "mdi:server-network", None, True),
-    "throughput_network_in": (
-        "Network throughput in",
-        DATA_RATE_MEGABYTES_PER_SECOND,
-        "mdi:server-network",
-        None,
-        True,
+    "throughput_network_out": SysMonitorSensorEntityDescription(
+        key="throughput_network_out",
+        name="Network throughput out",
+        native_unit_of_measurement=DATA_RATE_MEGABYTES_PER_SECOND,
+        icon="mdi:server-network",
+        state_class=STATE_CLASS_TOTAL,
+        mandatory_arg=True,
     ),
-    "throughput_network_out": (
-        "Network throughput out",
-        DATA_RATE_MEGABYTES_PER_SECOND,
-        "mdi:server-network",
-        None,
-        True,
+    "process": SysMonitorSensorEntityDescription(
+        key="process",
+        name="Process",
+        native_unit_of_measurement=" ",
+        icon=CPU_ICON,
+        state_class=STATE_CLASS_TOTAL,
+        mandatory_arg=True,
     ),
-    "process": ("Process", " ", CPU_ICON, None, True),
-    "processor_use": ("Processor use (percent)", PERCENTAGE, CPU_ICON, None, False),
-    "processor_temperature": (
-        "Processor temperature",
-        TEMP_CELSIUS,
-        None,
-        DEVICE_CLASS_TEMPERATURE,
-        False,
+    "processor_use": SysMonitorSensorEntityDescription(
+        key="processor_use",
+        name="Processor use (percent)",
+        native_unit_of_measurement=PERCENTAGE,
+        icon=CPU_ICON,
+        state_class=STATE_CLASS_TOTAL,
+    ),
+    "processor_temperature": SysMonitorSensorEntityDescription(
+        key="processor_temperature",
+        name="Processor temperature",
+        native_unit_of_measurement=TEMP_CELSIUS,
+        device_class=DEVICE_CLASS_TEMPERATURE,
+        state_class=STATE_CLASS_TOTAL,
+    ),
+    "swap_free": SysMonitorSensorEntityDescription(
+        key="swap_free",
+        name="Swap free",
+        native_unit_of_measurement=DATA_MEBIBYTES,
+        icon="mdi:harddisk",
+        state_class=STATE_CLASS_TOTAL,
+    ),
+    "swap_use": SysMonitorSensorEntityDescription(
+        key="swap_use",
+        name="Swap use",
+        native_unit_of_measurement=DATA_MEBIBYTES,
+        icon="mdi:harddisk",
+        state_class=STATE_CLASS_TOTAL,
+    ),
+    "swap_use_percent": SysMonitorSensorEntityDescription(
```

Comment:
Let's disable some by default. No need to have them all enabled by default.

Reply:
there is no sensor enabled by default, since the user needs to explicit enable them by set in configuration (see [docs](https://www.home-assistant.io/integrations/systemmonitor/))